src/index.html

```diff
@@ -11,12 +11,95 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;500&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        class="logo"
+        href="#"
+      >
+        <img
+          src="images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="list">
+          <li class="nav-link">
+            <a
+              class="link is-active"
+              href="#"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav-link">
+            <a
+              class="link"
+              href="#"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav-link">
+            <a
+              class="link"
+              data-qa="hover"
+              href="#"
+            >
+              LAPTOP & COMPUTERS
+            </a>
+          </li>
+          <li class="nav-link">
+            <a
+              class="link"
+              href="#"
+            >
+              GADGETS
+            </a>
+          </li>
+          <li class="nav-link">
+            <a
+              class="link"
+              href="#"
+            >
+              TABLES
+            </a>
+          </li>
+          <li class="nav-link">
+            <a
+              class="link"
+              href="#"
+            >
+              PHOTO
+            </a>
+          </li>
+          <li class="nav-link">
+            <a
+              class="link"
+              href="#"
+            >
+              VIDEO
+            </a>
+          </li>
```

Comment:
The class names should represent the meaning of the content, not the styles or tag names. Here, 'nav-link' seems to be more related to the style or the tag name. It would be better if you use a class name that represents the content of the list item. For example, if these are navigation menu items, you can name the class 'nav-item' or 'menu-item'. (HTML Content)